src/Facets/GenericSwapFacet.sol

```diff
@@ -11,8 +11,8 @@ import { InvalidReceiver } from "../Errors/GenericErrors.sol";
 
 /// @title GenericSwapFacet
 /// @author LI.FI (https://li.fi)
-/// @notice Provides functionality for swapping through ANY APPROVED DEX
-/// @dev Uses calldata to execute APPROVED arbitrary methods on DEXs
+/// @notice Provides functionality for swapping through ANY WHITELISTED DEX
```

Comment:
nitpick: I dont know why we are using all caps here, dont really  see a reason for that (same in V3)